features/crud.feature

```diff
@@ -358,6 +358,9 @@ Feature: Create-Retrieve-Update-Delete
       "alias": null
     }
     """
+  Scenario: PUT request on custom operations without content
```

Comment:
This scenario is incorrect. This is not a custom operation, and I don't suppose it should return 200 when the semantics is undefined (for the default PUT operation).